.travis.yml

```diff
@@ -6,10 +6,6 @@ addons:
   apt:
     packages:
       - libsundials-serial-dev
-branches:
```

Comment:
I think this was here to prevent PRs from being built twice (once as branch, once as PR)

Reply:
Yeah, unfortunately it has an undesirable side effect! We can turn off branch builds in the Travis web config, but then you don't end up with builds of master for readme badges etc I think. Or we could turn off normal branch builds and have a cron build of master each day?

Reply:
Hmmm... The `only: -master` is meant to make master build anyway. We use the same config without problems in PINTS

Reply:
Yeah, having that config worked for that. But what it broke was PRs only got built if they were against master, not if they were building on top of another branch.

Reply:
Oh yeah. I've seen that behaviour too!